python/3d/auto_generated/symbols/qgspointcloud3dsymbol.sip.in

```diff
@@ -68,6 +83,79 @@ Returns the point size of the point cloud
 Sets the point size
 
 .. seealso:: :py:func:`pointSize`
+%End
+
+    QgsPointCloud3DSymbol::RenderingStyle renderingStyle() const;
+%Docstring
+Returns the rendering style
+
+.. seealso:: :py:func:`setRenderingStyle`
+%End
+
+    void setRenderingStyle( QgsPointCloud3DSymbol::RenderingStyle style );
+%Docstring
+Sets the rendering style
+
+.. seealso:: :py:func:`renderingStyle`
+%End
+
+    QgsPointCloud3DSymbol::RenderingParameter renderingParameter() const;
+%Docstring
+Returns the parameter used to select the color of the point cloud when using color ramp coloring
+
+.. seealso:: :py:func:`setRenderingParameter`
+%End
+
+    void setRenderingParameter( QgsPointCloud3DSymbol::RenderingParameter parameter );
```

Comment:
would be better to allow this to be set as a string, so that users can visualise any of the attributes present in the data (eg. they may want to visual intensity or some other similar attribute)

Reply:
I was thinking about translation for this.
I seen from the ept.json file that there are many attributes along with some statistics about the attributes. I don't know how to access these now and it's useful to know them for the color ramp.

Reply:
Oh nevermind. I found them in QgsPointCloudLayer

Reply:
> with some statistics about the attributes.

I need access to these statistics too --  I plan to add them to the provider class at some stage.